MVCapplication/src/main/java/com/softserve/if072/mvcapp/dto/UserRegistrationForm.java

```diff
@@ -0,0 +1,59 @@
+package com.softserve.if072.mvcapp.dto;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class UserRegistrationForm {
```

Comment:
I got stuck with that. If we had single UserForm class with (email, password, name, role) fields, how could we implement two different validation strategies? I mean RegistrationController should have validated all the 4 fields, that's ok. But LoginController requires to have only two of them (email, password) validated. AFAIK there is no way to do that without implementing two different Validator instances (one for all 4 fields, and another for only 2 fields).
Obviously, two simple POJOs with annotation constraints are easier to maintain than one POJO plus two Validator classes :)